doc/GUIDE.md

```diff
@@ -498,75 +378,57 @@ someFunc :: IO ()
 
 ```
 michael@d30748af6d3d:~/helloworld$ stack build
-While constructing the BuildPlan the following exceptions were encountered:
-
---  While attempting to add dependency,
-    Could not find package acme-missiles in known packages
-
---  Failure when adding dependencies:
-      acme-missiles: needed (-any), stack configuration has no specified version (latest applicable is 0.3)
-    needed for package: helloworld-0.1.0.0
-
-Recommended action: try adding the following to your extra-deps in /home/michael/helloworld/stack.yaml
-- acme-missiles-0.3
-
-You may also want to try the 'stack solver' command
+# build failure output ...
 ```
 
-It says acme-missiles is "not present in build plan." This brings us to the next
-major topic in using stack.
+It says that it was unable to construct the build plan.
+
+This brings us to the next major topic in using stack.
 
 ## Curated package sets
 
-Remember above when `stack new` selected the lts-3.2 resolver for us? That
-defined our build plan and available packages. When we tried using the
-text package, it just worked, because it was part of the lts-3.2 *package set*.
-But acme-missiles is not part of that package set, so building failed.
+Remember above when `stack new` selected some [LTS resolver](https://github.com/fpco/lts-haskell#readme)
+for us? That defined our build plan and available packages. When we tried using the
+`text` package, it just worked, because it was part of the LTS *package set*.
+
+But `acme-missiles` is not part of that package set, so building failed.
 
 To add this new dependency, we'll use the `extra-deps` field in `stack.yaml` to
-define extra dependencies not present in the resolver. With that change, our
-`stack.yaml` looks like:
+define extra dependencies not present in the resolver. You can add this like so:
 
 ```yaml
-flags: {}
-packages:
-- '.'
 extra-deps:
-- acme-missiles-0.3 # not in lts-3.2
-resolver: lts-3.2
+- acme-missiles-0.3 # not in the LTS
 ```
 
 Now `stack build` will succeed.
 
 With that out of the way, let's dig a little bit more into these package sets,
-also known as *snapshots*. We mentioned lts-3.2, and you can get quite a bit of
-information about it at
-[https://www.stackage.org/lts-3.2](https://www.stackage.org/lts-3.2), including:
+also known as *snapshots*. We mentioned the LTS resolvers, and you can get quite a bit of
+information about it at [https://www.stackage.org/lts](https://www.stackage.org/lts), including:
 
-* The appropriate resolver value (`resolver: lts-3.2`, as we used above)
+* The appropriate resolver value (`resolver: lts-10.0`, as is currently the latest LTS)
 * The GHC version used
 * A full list of all packages available in this snapshot
 * The ability to perform a Hoogle search on the packages in this snapshot
-* A [list of all modules](https://www.stackage.org/lts-3.2/docs) in a snapshot,
+* A [list of all modules](https://www.stackage.org/lts-10.0/docs) in a snapshot,
```

Comment:
If we link to https://www.stackage.org/lts/docs that always seems to give the newest one. Is that worth doing, or think that'll be confusing?